### ISSUE TITLE
ProgressLabeller: Visual Data Stream Annotation for Training Object-Centric 3D Perception. (arXiv:2203.00283v1 [cs.RO])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#16275]
ProgressLabeller: Visual Data Stream Annotation for Training Object-Centric 3D Perception. (arXiv:2203.00283v1 [cs.RO])<br>
https://ift.tt/moWtic5<br>
<p>Visual perception tasks often require vast amounts of labelled data, including 3D poses and image space segmentation masks. The process of creating such training data sets can prove difficult or time-intensive to scale up to efficacy for general use. Consider the task of pose estimation for rigid objects. Deep neural network based approaches have shown good performance when trained on large, public datasets. However, adapting these networks for other novel objects, or fine-tuning existing models for different environments, requires significant time investment to generate newly labelled instances. Towards this end, we propose ProgressLabeller as a method for more efficiently generating large amounts of 6D pose training data from color images sequences for custom scenes in a scalable manner. ProgressLabeller is intended to also support transparent or translucent objects, for which the previous methods based on depth dense reconstruction will fail. We demonstrate the effectiveness of ProgressLabeller by rapidly create a dataset of over 1M samples with which we fine-tune a state-of-the-art pose estimation network in order to markedly improve the downstream robotic grasp success rates. ProgressLabeller will be made publicly available soon.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
